src/api/app/controllers/webui/repositories_controller.rb

```diff
@@ -17,7 +17,14 @@ def index
 
     @user_can_set_flags = policy(@project).update?
 
-    switch_to_webui2 if @package
```

Comment:
so you're breaking the consistency of bootstrap pages for packages? I don't like that idea.

Reply:
Good point! We can keep this line for now and once we have finished the repositories page for the projects, we will have only `switch_to_webui2` left.

Reply:
exactly, good catch @coolo :wink: It is changed